addons/account/models/partner.py

```diff
@@ -74,6 +77,19 @@ def _compute_foreign_vat_header_mode(self):
                 # 'no_template' kept for compatibility in stable. To remove in master
                 fiscal_position.foreign_vat_header_mode = 'templates_found' if template['installed'] else 'no_template'
 
+    @api.depends('tax_ids.tax_src_id', 'tax_ids.tax_dest_id')
+    def _compute_tax_map(self):
+        for position in self:
+            tax_map = defaultdict(list)
+            for tl in position.tax_ids:
+                tax_map[tl.tax_src_id.id].append(tl.tax_dest_id.id)
+            position.tax_map = dict(tax_map)
```

Comment:
I'm not sure in which context we care about creating from the UI?
Worst thing that can happen is that we will have a mapping to a `NewId` record in the binary field, but that is not an issue since it will never be read before it is saved